LEAF_Request_Portal/admin/templates/mod_workflow.tpl

```diff
@@ -413,6 +414,113 @@ function setInitialStep(stepID) {
     });
 }
 
+//list all action type to edit/delete
+function listActionType() {
+	dialog.hide();
+  $("#button_save").hide();
+	dialog.setTitle('List of Actions');
+	dialog.show();
+  $.ajax({
+		type: 'GET',
+		url: '../api/?a=workflow/userActions',
+		success: function(res) {
+			var buffer = '';
+			buffer += '<br /><table id="actions" border="1"><caption><h2>List of Current Actions:</h2></caption><tr><th scope="col">Action Name</th><th scope="col">Action</th></tr>';
```

Comment:
When showing Actions that are available to edit, it should also display the actions Past Tense name. 

And the "Action" column that contains the Edit/Delete buttons should not have a column name, calling it "Action" while editing Actions might be confusing to some Users.

Reply:
done